src/core/gui/dialog/backgroundSelect/BackgroundSelectDialogBase.cpp

```diff
@@ -102,12 +102,16 @@ void BackgroundSelectDialogBase::setSelected(size_t selected) {
     }
 
     size_t lastSelected = this->selected;
-    if (lastSelected < elements.size()) {
-        elements[lastSelected]->setSelected(false);
+    if (lastSelected < entries.size()) {
+        entries[lastSelected]->setSelected(false);
     }
 
-    if (selected < elements.size()) {
-        elements[selected]->setSelected(true);
+    if (selected < entries.size()) {
+        entries[selected]->setSelected(true);
         this->selected = selected;
     }
+
+    if (selected < this->entries.size()) {
+        gtk_widget_set_sensitive(okButton, true);
+    }
```

Comment:
Can't this be simplified as follows?
```suggestion
    if (selected < entries.size()) {
        entries[selected]->setSelected(true);
        this->selected = selected;
        gtk_widget_set_sensitive(okButton, true);
    }
```